recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-core/images/core-image-thin-initramfs.bbappend

```diff
@@ -1,4 +1,4 @@
-DEPENDS += "u-boot-mkimage-native"
+DEPENDS += "u-boot u-boot-mkimage-native"
```

Comment:
This effectively means that EVERY build will have u-boot built, e.g. even if we do not build DomA

Reply:
replaced at ${@bb.utils.contains('XT_GUESTS_INSTALL', 'doma', 'u-boot', '', d)} \

Reply:
can't see the change